tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/impl/PostServiceImpl.java

```diff
@@ -666,12 +666,12 @@ public PostDTO findOne(Long id) {
   @Override
   public void delete(Long id) {
     log.debug("Request to delete Post : {}", id);
+    checkIfReconciledWithESR(id);
```

Comment:
I'm fairly sure this will need to be changed & done differently.
Please discuss this, using checks / warnings elsewhere in the system, where we display something to users before they modify data.